generic/tests/trans_hugepage_relocated.py

```diff
@@ -21,35 +24,37 @@ def run(test, params, env):
     :param params: Dictionary with test parameters.
     :param env: Dictionary with the test environment.
     """
+
     def nr_hugepage_check(sleep_time, wait_time):
         time_last = 0
         while True:
             value = int(utils_memory.read_from_meminfo("AnonHugePages"))
+            logging.debug("The AnonHugePages value: %d", value)
             nr_hugepages.append(value)
             time_stamp = time.time()
             if time_last != 0:
                 if nr_hugepages[-2] != nr_hugepages[-1]:
                     time_last = time_stamp
                 elif time_stamp - time_last > wait_time:
-                    test.log.info("Huge page size stop changed")
+                    logging.info("Huge page size stop changed")
```

Comment:
`test.log` is the recommended method,  if there any reason to change it to `logging` ?

Reply:
This patch is based in the previous draft: https://github.com/autotest/tp-qemu/pull/3132 but for today it could be outdated :grinning: I will change to `test.log`

Reply:
Updated !